ImportExport/Processor/ProductImageImportProcessor.php

```diff
@@ -83,7 +84,7 @@ private function mergeImages(Product $product, array $images): Product
             }
 
             if (!is_a($image->getImage()->getParentEntityClass(), ProductImage::class, true)) {
-                $image->setImage(null);
```

Comment:
I think that was a fix for old bug with duplicate images, we don't have it for a while so we can remove this condition